op-e2e/op_geth_test.go

```diff
@@ -38,6 +38,7 @@ func TestMissingGasLimit(t *testing.T) {
 	attrs.GasLimit = nil
 
 	res, err := opGeth.StartBlockBuilding(ctx, attrs)
```

Comment:
I'd been wary that these methods were doing something fancier, but looks like it's simply depending on the RPC link between the l2engine and the client over RPC, that works out nicely!

Reply:
Yup!